client/src/routes/MusicMaker/Octave.tsx

```diff
@@ -1,176 +1,173 @@
+              onClick={() => keyBoardPress("D3")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("D#3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("E3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("F3")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("F#3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("G3")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("G#3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("A3")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("A#3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("B3")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("C4")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("C#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("D4")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("D#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("E4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("F4")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("F#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("G4")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("G#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("A4")}
+            ></div>
+            <div
+              className="key key-sharp"
+              onClick={() => keyBoardPress("A#4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("B4")}
+            ></div>
+            <div
+              className="key key-natural"
+              onClick={() => keyBoardPress("C5")}
+            ></div>
```

Comment:
this repetition defeats the purpose of React. how can you write this to be DRY (Don't Repeat Yourself)?

Reply:
done!